src/main/webapp/script.js

```diff
@@ -1,33 +1,26 @@
-function ratePromoCode(tableRow, rating) {
-    var videoUrl = document.getElementById('promoCodeTable').rows[tableRow].cells[0].firstChild.getAttribute("href");
-    fetch('/promo-code?rating=' + rating + '&videoUrl=' + videoUrl, {method: 'POST'});
-    displayCodes();
-}
+//This function is not currently in use but may be used in further implementations
+// function ratePromoCode(tableRow, rating) {
+//     var videoUrl = document.getElementById('promoCodeTable').rows[tableRow].cells[0].firstChild.getAttribute("href");
+//     console.log(videoUrl);
+//     fetch('/promo-code?rating=' + rating + '&videoUrl=' + videoUrl, {method: 'POST'});
+//     getCodes();
+// }
```

Comment:
Delete this function, version control will remember it if you need it for future implementation. Also explained in Clean Code chapter 4 section "Commented-Out Code" (pg. 68)